package-lock.json

```diff
@@ -7211,4 +7211,4 @@
       }
     }
   }
-}
+}
```

Comment:
please remove this modification

Reply:
Ignore package-lock.json?

Reply:
remove it from this PR (don't delete the file)

Reply:
Still...

Reply:
It is removed